Examples/ocaml/simple/index.html

```diff
@@ -57,37 +57,37 @@
 <h2>Compilation</h2>
 
 <ol>
-<li><tt>swig -python <a href="example.i">example.i</a></tt>
+<li><tt>swig -ocaml <a href="example.i">example.i</a></tt>
 <p>
 <li>Compile <tt><a href="example_wrap.c">example_wrap.c</a></tt> and <tt><a href="example.c">example.c</a></tt>
 to create the extension <tt>examplemodule.so</tt>. 
 </ol>
 
 <h2>Using the extension</h2>
 
-Click <a href="example.py">here</a> to see a script that calls our C functions from Python.
+Click <a href="example.ml">here</a> to see a script that calls our C functions from Ocaml.
 
 <h2>Key points</h2>
 
 <ul>
-<li>Use the <tt>import</tt> statement to load your extension module from Python. For example:
+<li>Use the <tt>open</tt> statement to load your extension module from Ocaml. For example:
 <blockquote>
 <pre>
-import example
+open Example
 </pre>
 </blockquote>
 
-<li>C functions work just like Python functions. For example:
+<li>C functions work just like Ocaml functions. For example:
 <blockquote>
 <pre>
-g = example.gcd(42,105)
+let g = _gcd '(x,y) as int
 </pre>
 </blockquote>
 
-<li>C global variables are accessed through a special variable called 'cvar'. For example:
+<li>C global variables are accessed through a special variable called '_Foo'. For example:
 <blockquote>
 <pre>
-a = example.cvar.Foo
+let _ = Printf.printf "Foo = %f\n" (_Foo '() as float)
```

Comment:
The "through a special variable called" part doesn't seem the right wording for ocaml - in python `Bar` maps to `example.cvar.Bar`.  I'm not familiar with the ocaml backend, but in ocaml `Bar` presumably isn't accessed through `_Foo` - probably this should say something like "C global variable `Foo` is wrapped as `_Foo` in ocaml"?